src/App.css

```diff
@@ -1,29 +1,65 @@
-.app {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  height: 100vh;
+/* stylelint-disable declaration-block-no-shorthand-property-overrides */
```

Comment:
- [ ] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉

Reply:
thanks very much